src/Pages/NewListingPage.tsx

```diff
@@ -1,31 +1,51 @@
-import React, { FormEvent, useState } from "react";
+import React, { CSSProperties, FormEvent, useState } from "react";
```

Comment:
Can you remove the unused imports? CSS, Formevent, iconbutton, snackbar, closeicon, formik and Ilistitem?